src/Plugin/WebformElement/NetsEasyPaymentElement.php

```diff
@@ -116,14 +112,20 @@ public function alterForm(array &$element, array &$form, FormStateInterface $for
        * inject placeholder for nets gateway containing amount to pay.
        */
       if ($amount_to_pay > 0) {
-        $form['content']['#markup'] = $element['#checkout_page_description'] ?? NULL;
+        if (!empty($element['#checkout_page_description'])) {
+          $form['content']['#markup'] = $element['#checkout_page_description'];
```

Comment:
You should use the module name in the key to minimize the probability of accidentally overwriting an existing key:

```suggestion
          $form['os2forms_payment_content']['#markup'] = $element['#checkout_page_description'];
```

Do the same for `checkout_container` below.